app/uk/gov/hmrc/nisp/controllers/NIRecordController.scala

```diff
@@ -44,6 +45,7 @@ trait NIRecordController extends NispFrontendController with AuthorisedForNisp w
   val nispConnector: NispConnector
   val customAuditConnector: CustomAuditConnector
   val showFullNI: Boolean
+  val currentDate = new LocalDate(DateTimeZone.forID("Europe/London"))
```

Comment:
Sorry, what I mean is, it's need to be implemented like this in the object, only declared in the trait.